vera_bench/vera_runner.py

```
@@ -0,0 +1,126 @@
+            tier3_runtime=verification.get("tier3_runtime", 0),
+            total=verification.get("total", 0),
+            diagnostics=data.get("diagnostics", []),
+            warnings=data.get("warnings", []),
+        )
+
+    def run_fn(self, file_path: str | Path, fn_name: str, args: list | None = None) -> RunResult:
+        cmd = [self.vera, "run", str(file_path), "--fn", fn_name]
+        if args:
+            cmd.append("--")
+            cmd.extend(str(a) for a in args)
+        result = subprocess.run(
```

Ruff (0.15.7)

[error] `subprocess` call: check for execution of untrusted input

(S603)